pkg/build/buildutil/buildutil.go

```diff
@@ -1,9 +1,21 @@
 package buildutil
 
 import (
+	"path/filepath"
+	"strings"
+
 	corev1 "k8s.io/api/core/v1"
+	kvalidation "k8s.io/apimachinery/pkg/util/validation"
 
 	buildv1 "github.com/openshift/api/build/v1"
+	"github.com/openshift/library-go/pkg/build/naming"
```

Comment:
nit - needs to be in a separate import block, since this references a local package?
```suggestion

	"github.com/openshift/library-go/pkg/build/naming"
```